### ISSUE TITLE
Calibration preview images not appearing

### ISSUE THREAD

**Issue by micahsj00** [dicengine/dice#215]
Hi,
When I go to calibrate images that I took, no preview of the images shows up. When I use the slider to go through the images, the green box will appear, but not the images themselves. I calibrated this set of images a few days ago with no issues and the images preview showed up then. When I try with other sequences of images, the preview also doesn't show up. Changing if the threshold is included or not doesn't change anything, and manually setting the threshold also doesn't make the image preview appear. The calibration console looks normal and it isn't saying there are any errors or issues. Everything else seems to be working fine, and the error values I'm getting seem right based on how/if I change the threshold. On auto threshold I think the error values are the same as the values I got when I did the calibration on these images a few days ago, although I'm just going off my memory on that so I can't be certain. Here's a screenshot of what it looks like.
![DICe cal preview issue](https://user-images.githubusercontent.com/59046842/124662269-97d8b100-de76-11eb-8ec7-7087844763f1.PNG)

Please let me know what I should try to fix this. My error values (especially RMS error) are higher than I want them to be, but without being able to see the preview images I can't look at them to improve that.

Thanks.

**Comment by dicengine** [comment]
That is very strange. Can you look in the .dice folder for the working directory and see if there is a .preview_cal_left.png or a .preview_cal_right.png file? You should be able to open either of these with any photo viewer.

**Comment by micahsj00** [comment]
Yes, I see those files for both the left and right and can open and view them with my photo viewer. For both it's the last image in the sequence.

**Comment by dicengine** [comment]
It's still strange to me that you can't see the images. Try deleting the cal preview images mentioned above and see if that helps. You could also re-install DICe to see if that works. Otherwise I'll have to do some debugging to see if I can re-create the issue your having.

**Comment by micahsj00** [comment]
I tried deleting the images and that didn't work, nor did re-installing. I did however figure out a work around that I can use for now. By opening the .dice folder and making the icons extra large, I can get a pretty good view of the images. And when I move the preview slider I can see the image numbers that I want, and the threshold shows up if I include it.
